app/backend/src/couchers/models.py

```diff
@@ -1058,3 +1068,14 @@ class Reply(Base):
     deleted = Column(DateTime(timezone=True), nullable=True)
 
     comment = relationship("Comment", backref="replies")
+
+
+class LanguageAbilities(Base):
+    __tablename__ = "language_abilities"
+
+    id = Column(BigInteger, primary_key=True)
+    language = Column(String(length=3), nullable=False)
```

Comment:
Please add a comment on the language coding; `# ISO639-3 code lowercase`

Reply:
done